src/ref/ui-patterns/web/lightbox-image.md

```diff
@@ -40,5 +40,15 @@ To add rounded corners to images inside LightBoxImage, add the following custom
   border-radius: var(--border-radius-soft);
 }
 ```
+<<<<<<< HEAD
```

Comment:
There's still a conflict here!
Please remove these lines (my change was to remove all the commented lines).